great_expectations/data_context/store/database_store_backend.py

```diff
@@ -115,13 +117,7 @@ def _build_engine(self, credentials, **kwargs) -> "sa.engine.Engine":
         """
         # Update credentials with anything passed during connection time
         drivername = credentials.pop("drivername")
-        schema_name = credentials.pop("schema_name", None)
-        if schema_name is not None:
-            logger.warning(
-                "schema_name specified creating a URL with schema is not supported. Set a default "
-                "schema on the user connecting to your database."
-            )
-
+        self._schema_name = credentials.pop("schema_name", None)
```

Comment:
Should this be above line 79 where it is first referenced?

Reply:
You're absolutely right :) Thanks for catching this